WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3493,6 +3493,9 @@ public void onMediaDeleted(String localMediaId) {
                     return;
                 }
 
+                mAztecBackspaceDeletedOrGbBlockDeletedMediaItemIds.add(localMediaId);
```

Comment:
cc:reuse

Reply:
addressed in 133db73583 (same as above) 👍